packages/edit-site/src/components/sidebar/style.scss

```diff
@@ -136,3 +136,10 @@ $split-border-control-offset: 55px;
 		}
 	}
 }
+
+// Override the `hr` styles defined for the complementary area component
+// from the `@wordpress/interface` package.
+.edit-site-global-styles-sidebar hr {
+	margin: 0;
+	border-color: rgba(0, 0, 0, 0.1);
```

Comment:
Hoping we can replace this color value with a CSS var in the near future 🤞 

(Leaving a breadcrumb here for future cleanup #40392)

Reply:
We use `$gray-200` for similar borders, but I sense there may be a different effort at work here?

Reply:
I will remove this line (as explained in https://github.com/WordPress/gutenberg/pull/40526#discussion_r856006044).

We can continue the conversation there, regarding which shade of gray should be used going forward